.gitignore

```diff
@@ -2,6 +2,8 @@ node_modules
 /out
 src/raw/bower_components
 tmp
+deploy-key
```

Comment:
You are right. Still you do not add ssh-key to .gitignore. You should do it if you did it with deploy-key. For me it can stay.